packages/react/src/components/SwatchColorPicker/SwatchColorPicker.base.tsx

```diff
@@ -109,6 +109,10 @@ export const SwatchColorPickerBase: React.FunctionComponent<ISwatchColorPickerPr
     focusedContainer: classNames.focusedContainer,
   };
 
+  // if there is only one row of cells, they should use radio semantics
```

Comment:
Can you add a little more about why? Having the explanation inline in theory should make the fix more persistent. :P